src/main/groovy/com/mx/coppuccino/CoppuccinoPluginExtension.groovy

```diff
@@ -28,13 +28,6 @@ class CoppuccinoCoverageExtension {
   ]
 }
 
-class CoppuccinoDependenciesExtension {
-  // Enable Dependency locking on all configurations
-  boolean lockingEnabled = true
```

Comment:
will Hush enabled dependency locking and handle prerelease versions?

Reply:
No, it definitely won't. Sounds like I need to add that 🤔

Reply:
Yeah, I think preserving the behavior here would be fine too. This is unrelated to CVE scanning, and more related to Gradle config.

Reply:
Oh cool. I'll refactor this a bit then, thank you!